packages/email-field/src/vaadin-email-field.js

```diff
@@ -62,6 +63,9 @@ export class EmailField extends TextField {
     if (this.inputElement) {
       this.inputElement.autocapitalize = 'off';
     }
+
+    this._tooltipController = new TooltipController(this);
+    this.addController(this._tooltipController);
```

Comment:
We shouldn't need to create a controller here, as it's inherited from `TextField`.

Reply:
Done.